app/ui/header/header.tsx

```diff
@@ -10,7 +10,7 @@ const Header = () => {
     <header className="py-10">
       <div className="flex mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
         <h1 className="text-3xl font-bold tracking-tight text-white flex-grow">
-          Dashboard
+          Database
```

Comment:
lets come up with something other than database. The main view is going to show a list of records. Maybe 'Records' would work. The functionality we want is to be able to tag and push records to external systems. Maybe we call it "Tagger", or "Tag Manager"?